.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -54,7 +58,9 @@ Read the release notes for full details: {{url}}
 
 Following our support policy, versions {{versions}} are no longer supported. Projects running these versions should start planning to upgrade in the near future. For more details read our software support documentation: https://docs.communityhealthtoolkit.org/core/overview/supported-software/
 
-To see what's scheduled for the next releases have a read of the product roadmap: https://github.com/orgs/medic/projects?query=is%3Aopen+sort%3Aname-asc
+See what's scheduled for the next releases: https://github.com/orgs/medic/projects?query=is%3Aopen+sort%3Aname-asc
 ```
-- [ ] Announce the release on the [CHT forum](https://forum.communityhealthtoolkit.org/), under the "Product - Releases" category. You can use the previous message and omit `@channel`.
+- [ ] Announce the release on the [CHT forum](https://forum.communityhealthtoolkit.org/c/product/releases/26), under the "Product - Releases" category. You can use the previous message and omit `@channel`.
+- [ ] Announce the release of new documenation on the [CHT Forum](https://forum.communityhealthtoolkit.org/c/product/documentation/28), under the "Product - Documentation" category.
```

Comment:
Can you give an example of what this should look like, so the release manager has something to work from? How will the release manager find "new documentation"? Shouldn't the new documentation be released and announced incrementally rather than all in one go when the product is released, or is this just for documentation of new features in the release?

Reply:
Updated verbage to reflect documentation that pertains to features and improvements in the release. This is a new part of the process so no example to point to yet. Once I do one, I will add a template.